core/src/main/scala/kafka/server/KafkaConfig.scala

```diff
@@ -88,6 +88,8 @@ object Defaults {
   val MaxConnections: Int = Int.MaxValue
   val ConnectionsMaxIdleMs = 10 * 60 * 1000L
   val RequestTimeoutMs = 30000
+  val ConnectionSetupTimeoutMs = 10 * 1000L
+  val ConnectionSetupTimeoutMaxMs = 127 * 1000L
```

Comment:
We can use the ones defined in `CommonClientConfigs` here.

Reply:
Yes. Refactored.